projects/plugins/jetpack/_inc/client/at-a-glance/backups.jsx

```diff
@@ -518,6 +533,8 @@ class DashBackups extends Component {
 export default connect(
 	state => {
 		return {
+			sitePlan: getSitePlan( state ),
+			blogID: getSiteId( state ),
```

Comment:
Can we please use `getSiteData().wpcom` from `@automattic/jetpack-script-data` to get them blog ID?